packages/grafana-ui/src/components/ValuePicker/ValuePicker.story.tsx

```diff
@@ -15,33 +15,42 @@ export default {
     docs: {
       page: mdx,
     },
+    knobs: {
+      disabled: true,
+    },
+    controls: {
+      exclude: ['onChange', 'options'],
+    },
+  },
+  argTypes: {
+    variant: {
+      control: {
+        type: 'select',
+        options: ['primary', 'secondary', 'destructive', 'link'],
+      },
+    },
+    icon: {
+      control: {
+        type: 'select',
+        options: getAvailableIcons(),
+      },
+    },
   },
 };
-const VISUAL_GROUP = 'Visual options';
-const variants = ['primary', 'secondary', 'destructive', 'link'];
-const sizes = ['sm', 'md', 'lg'];
 const options = generateOptions();
 
-export const simple = () => {
-  const label = text('Label', 'Pick an option', VISUAL_GROUP);
-  const variant = select('Variant', variants, 'primary', VISUAL_GROUP);
-  const size = select('Size', sizes, 'md', VISUAL_GROUP);
-  const isFullWidth = boolean('Is full width', false, VISUAL_GROUP);
-  const icon = getIconKnob();
-  const menuPlacement = select('Menu placement', ['auto', 'bottom', 'top'], 'auto', VISUAL_GROUP);
-
+export const Simple: Story<ValuePickerProps<T>> = (args) => {
```

Comment:
`<T>` gives off this error: Cannot find name 'T'. I tried removing T completely but I figured it's not the best approach. Do you have any idea of how we could solve this? @Clarity-89

Reply:
`T` in this case points to the value of the option object, which is a `string`.